src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("user");
+        user.setPassword("psswrd");
+        user.setAge(18);
+    }
+
+    @Test
+    void register_fullSuccess_ok() {
+        User testUser = registrationService.register(user);
+        assertEquals(user, testUser);
+    }
+
+    @Test
+    void register_existUser_notOk() {
+        Storage.people.add(user);
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_setEmptyLogin_notOk() {
```

Comment:
```suggestion
    void register_emptyLogin_notOk() {
```
or
```suggestion
    void register_userWithEmptyLogin_notOk() {
```
The main point of the central part of test method name is to describe input data